src/symbolize.cc

```diff
@@ -837,6 +837,57 @@ static ATTRIBUTE_NOINLINE bool SymbolizeAndDemangle(void *pc, char *out,
 
 _END_GOOGLE_NAMESPACE_
 
+#elif defined(OS_WINDOWS)
+
+#include "Dbghelp.h"
+
+_START_GOOGLE_NAMESPACE_
+
+class SymInitializer {
```

Comment:
This class should not be copied, correct? To ensure no one does this in the future, I'd delete the copy constructor and copy assignment operator by adding the declarations
```c++
SymInitializer(const SymInitializer&);
SymInitializer& operator=(const SymInitializer&);
```
to the private section of the class.